assets/js/editor-components/sidebar-compatibility-notice/style.scss

```diff
@@ -0,0 +1,13 @@
+.wc-blocks-sidebar-compatibility-notice {
+	border-top: 1px solid $gray-200;
+}
+
+.wc-blocks-sidebar-compatibility-notice__notice.components-notice.is-dismissible {
+	margin: 0;
+	padding: $gap;
+	padding-right: $gap-small;
```

Comment:
We have more `padding` to the right which produces an uneven spacing.

<img width="306" alt="image" src="https://user-images.githubusercontent.com/14235870/183070154-50fa063f-58cf-42ce-959c-80b013abd5e7.png">